src/Html.php

```diff
@@ -5547,21 +5547,18 @@ class='form-control'
                       . ($p['multiple'] ? " multiple='multiple'" : "")
                       . ($p['onlyimages'] ? " accept='.gif,.png,.jpg,.jpeg'" : "") . ">";
 
-        $progressall_js = '';
-        if (!$p['only_uploaded_files']) {
```

Comment:
from docblock 
```
only_uploaded_files boolean  show only the uploaded files block, i.e. no title, no dropzone
     *                                   (should be false when upload has to be enable only from rich text editor)
```

then we can safely consider that the success / error feedback area should be rendered no matter the drop zone is rendered or not.